DOCS/git-usage.md

```diff
@@ -0,0 +1,104 @@
+# Git usage guidance
+
+## Purpose
+
+Some contributors to the SPDX License List may be unfamiliar with Git command-line usage and practices. The GitHub UI enables contributions without needing to use the command line, and this is sufficient for many edits.
+
+However, using Git via the command line can be more efficient, and can make it easier to test your license templates and edits before submitting a PR.
+
+This document provides some guidance for setting up and using Git on your own system to contribute to the SPDX License List on GitHub. Suggestions and edits to this document are greatly welcomed!
+
+## Prerequisites
+
+This guide assumes that you've already done the following:
+
+* set up a GitHub account
+* forked the **upstream** repo (https://github.com/spdx/license-list-XML) into your own **personal** repo (e.g., https://github.com/swinslow/license-list-XML)
+* your forked repo's master branch should currently be in-line with the upstream repo.
+  * In other words, the upstream repo and your repo should both have the same "latest commit" ID: ![screenshot of commit ID in GitHub UI](/DOCS/images/git-usage-commit-id.png)
+  * If the commit IDs are different, you can use the UI to fetch the differences from upstream into your repo: ![screenshot of fetch upstream option in GitHub UI](/DOCS/images/git-usage-fetch-upstream-ui.png)
+* opened a text terminal program
+  * Linux: if you aren't sure, look for a program called something like "Terminal", "xterm" or "gnome-terminal"
+  * MacOS: there is a built-in text terminal at /Applications/Utilities/Terminal
+  * Windows: _to be filled in by someone else with Windows terminal experience_
+* [installed Git](https://github.com/git-guides/install-git)
```

Comment:
```suggestion
* [installed Git](https://github.com/git-guides/install-git)
* opened a text terminal program
  * Linux: if you aren't sure, look for a program called something like "Terminal", "xterm" or "gnome-terminal"
  * MacOS: there is a built-in text terminal at /Applications/Utilities/Terminal
  * Windows: start the 'Git Bash' program, which you should be able to find in your Start Menu
```

For Windows, Git will need to have been installed beforehand so that Git Bash is available. One *could* use Command Prompt instead of Git Bash, but I don't think we'd want to inflict that upon anybody! :stuck_out_tongue:

Reply:
Great suggestion -- thanks, I didn't know how this worked on Windows  :)